viewer/packages/cad-parsers/src/metadata/parsers/types.ts

```diff
@@ -14,9 +14,7 @@ export type CadSceneRootMetadata = {
   // readonly subRevisionId: number;
 };
 
-export type SceneSectorMetadata = BaseSceneSectorMetadata & (GltfSceneSectorMetadata | I3dF3dSceneSectorMetadata);
-export type V8SceneSectorMetadata = BaseSceneSectorMetadata & I3dF3dSceneSectorMetadata;
-export type V9SceneSectorMetadata = BaseSceneSectorMetadata & GltfSceneSectorMetadata;
+export type SceneSectorMetadata = BaseSceneSectorMetadata & GltfSceneSectorMetadata;
```

Comment:
Is there a point in splitting this into `BaseSceneSectorMetadata` and `GltfSceneSectorMetadata` anymore?